package.json

```diff
@@ -101,7 +101,7 @@
     "react-native-sound": "0.11.2",
     "react-native-splash-screen": "3.3.0",
     "react-native-svg": "13.13.0",
-    "react-native-svg-transformer": "1.1.0",
+    "react-native-svg-transformer": "1.2.0",
```

Comment:
This update fixed this issue 
https://github.com/kristerkari/react-native-svg-transformer/issues/317

Reply:
Great!